### PR TITLE
[OTX][Hot-fix] Add init_weights call in the mmcls builder.py

### PR DESCRIPTION
**Pull request by JihwanEom**
Same bug with #1554, but the classifier head was not initialized until now. (not caused by public mmcv PR #1497 )

### CODE REVIEW

File: otx/algorithms/classification/adapters/mmcls/utils/builder.py [hunk 1]
```diff
@@ -31,6 +35,9 @@ def build_classifier(
 
     model_cfg = deepcopy(config.model)
     model = origin_build_classifier(model_cfg)
+    logger.setLevel("WARNING")
+    model.init_weights()
```
Comment (harimkang) [inline otx/algorithms/classification/adapters/mmcls/utils/builder.py:39]: What is the difference between `init_weight` and `load_checkpoint` (line 45)?

**Comment by harimkang** [comment]
Shouldn't this be segmentation as well?

File: otx/algorithms/classification/adapters/mmcls/utils/builder.py (hunk 1 above)
Comment (JihwanEom) [reply otx/algorithms/classification/adapters/mmcls/utils/builder.py:39]: `init_weight` calls initialization methods like `Normal init` or `Linear init`. `load_checkpoint` loads incoming checkpoint from Imagenet or `load-from` argument.

**Review by harimkang — DISMISSED** [review]
LGTM

**Review by supersoob — CHANGES_REQUESTED** [review]
I don't think it's good to add the init_weights for seg in this PR. Plus, It already calls it inside the model backbone and heads. If you adds the line, it is redundantly added it and produces the redundant logs. Please revert it @harimkang @JihwanEom

**Comment by JihwanEom** [comment]
> 

Could you check again with log? `decode_head.conv_seg` is not initialized properly. Also, there will be no additional logs because the log level has been updated.

image

**Comment by supersoob** [comment]
> > 
> 
> Could you check again with log? `decode_head.conv_seg` is not initialized properly.
> 
> image

I don't understand what you want me to check and why you think it is not initialized properly. init_weights does not load load_from arg, what we actually want to load.

**Comment by harimkang** [comment]
> I don't think it's good to add the init_weights for seg in this PR. Plus, It already calls it inside the model backbone and heads. If you adds the line, it is redundantly added it and produces the redundant logs. Please revert it @harimkang @JihwanEom

@jaegukhyun I think this is no different from detection. Do you have any advice?

**Comment by JihwanEom** [comment]
I'll revert segmentation part, let's talk on other PR for segmentation.

**Comment by JihwanEom** [comment]
> I don't think it's good to add the init_weights for seg in this PR. Plus, It already calls it inside the model backbone and heads. If you adds the line, it is redundantly added it and produces the redundant logs. Please revert it @harimkang @JihwanEom

When I add `init_weight` in builder.py, the accuracy was improved (41.75 -> 45.13 on mbv3). Check it out next time when you have a room. Sorry for inconvinience.

**Review by supersoob — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Pull request merged**